src/Components/Publishing/Series/ArticleCard.tsx

```diff
@@ -0,0 +1,144 @@
+  series?: any
+}
+
+export class ArticleCard extends Component<Props, null> {
+  public static defaultProps: Partial<Props>
+
+  render () {
+    const { article, color, series } = this.props
+    const { video } = article
+
+    return (
+      <ArticleCardContainer href={article.slug} color={color} className="ArticleCard">
+
+        <TextContainer>
+          <div>
+            <Header>
+              <div>{series.title}</div>
+              {video &&
+                <div>{video.length}</div>
+              }
+            </Header>
+            <Title>{article.title}</Title>
+            <Description>{article.description}</Description>
+          </div>
+          <Byline
+            article={article}
+            color={color}
+            layout='condensed'
+          />
+        </TextContainer>
+
+        <ImageContainer>
+          <Image src={crop(article.thumbnail_image, { width: 680, height: 450 })} />
+          {video &&
+            <VideoPlay />
+          }
+        </ImageContainer>
+
+      </ArticleCardContainer>
+    )
+  }
+}
+
+ArticleCard.defaultProps = {
+  color: 'black'
+}
+
+const A: StyledFunction<Props & React.HTMLProps<HTMLLinkElement>> = styled.a
+const Div: StyledFunction<Props & React.HTMLProps<HTMLDivElement>> = styled.div
+
+const ArticleCardContainer = A`
+  display: block;
+  border: 1px solid;
+  color: ${props => props.color};
+  text-decoration: none;
+  padding: 30px;
+  display: flex;
+  ${props => pMedia.md`
+    flex-direction: column-reverse;
+    padding: 20px;
+  `}
+`
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 50%;
+  margin-bottom: 5px;
+  .author, .date {
+    ${Fonts.unica("s16", "medium")}
+  }
+  ${props => pMedia.md`
+    width: 100%;
+    margin-bottom: 0;
+  `}
+`
+
+const Title = styled.div`
+  ${Fonts.unica("s45")}
+  margin-bottom: 30px;
+  ${props => pMedia.md`
+    ${Fonts.unica("s32")}
+  `}
+`
+
+const Header = styled.div`
+  ${Fonts.unica("s16", "medium")}
+  display: flex;
+  margin-bottom: 10px;
+  div {
+    margin-right: 20px;
+  }
+`
+const Description = styled.div`
+  ${Fonts.garamond("s23")}
+  ${props => pMedia.md`
+    ${Fonts.garamond("s19")}
+    margin-bottom: 20px;
+  `}
+`
+
+const ImageContainer = styled.div`
+  position: relative;
+  width: 50%;
+  height: 100%;
+  margin-left: 30px;
+  ${props => pMedia.md`
+    width: 100%;
+    margin-left: 0;
+    margin-bottom: 10px;
+  `}
+`
+
+const Image = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
+`
+
+const VideoPlay = Div`
+  position: absolute;
+  left: 20px; 
+  bottom: 20px;
+  border-top: 30px solid transparent;
+  border-bottom: 30px solid transparent;
+  border-left: 40px solid ${props => props.color};
+  ${props => pMedia.md`
+    border-top: 22px solid transparent;
+    border-bottom: 22px solid transparent;
+    border-left: 30px solid ${props.color};
+  `}
+`
```

Comment:
Ah the play button has been implemented a few times in Reaction now - might want to consider extracting this out into a reusable styled div or as an SVG?

Reply:
We already have a component for video play, but left this one as css because it has no 'onPlay' triggering.

Reply:
SVG is probably a good idea, though the width here is a little different from our normal caret -- will check w owen if we can standardize.